Library/Formula/gtk-mac-integration.rb

```diff
@@ -0,0 +1,156 @@
+class GtkMacIntegration < Formula
+  homepage "https://wiki.gnome.org/Projects/GTK%2B/OSX"
+  url "http://ftp.gnome.org/pub/gnome/sources/gtk-mac-integration/2.0/gtk-mac-integration-2.0.7.tar.xz"
+  sha256 "e6de70da740b452f18c113a7ebb6772936a0fb9873137514090849820e296eb7"
+
+  depends_on "pkg-config" => :build
+  depends_on "xz" => :build
+  depends_on "cairo" => "without-x11"
+  depends_on "pango" => "without-x11"
+  depends_on "gtk+" => "without-x11"
+  depends_on "pygtk" => :optional
+
+  head do
+    url "git://git.gnome.org/gtk-mac-integration"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool" => :build
+    depends_on "gtk-doc" => :build
+  end
+
+  stable do
+    patch :DATA
+  end
+
+  def install
+    if build.head?
+      ENV["LIBTOOL"] = "glibtool"
+      ENV["LIBTOOLIZE"] = "glibtoolize"
+      system "NOCONFIGURE=1", "./autogen.sh"
+    end
+    system "./configure", "--prefix=#{prefix}",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--with-gtk=gtk+-2.0"
+    system "make", "install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!